tests/openedx_tagging/core/tagging/test_models.py

```diff
@@ -220,11 +242,54 @@ def test_get_tags(self):
             *self.phylum_tags,
         ]
 
+    def test_get_root_tags(self):
+        assert list(self.taxonomy.get_filtered_tags()) == self.domain_tags
+        assert list(
+            self.taxonomy.get_filtered_tags(search_term='aR')
+        ) == self.filtered_domain_tags
+
     def test_get_tags_free_text(self):
         self.taxonomy.allow_free_text = True
         with self.assertNumQueries(0):
             assert self.taxonomy.get_tags() == []
 
+    def test_get_children_tags(self):
+        assert list(
+            self.taxonomy.get_filtered_tags(parent_tag_id=self.animalia.id)
+        ) == self.phylum_tags
+        print(self.taxonomy.get_filtered_tags(
+                parent_tag_id=self.animalia.id,
+                search_term='dA',
+            ))
+        print(self.filtered_phylum_tags)
```

Comment:
`print` statements left in by accident, I assume?

Reply:
Fixed here: https://github.com/openedx/openedx-learning/pull/78/commits/9a15562c8509eee795dff0a939a9310b8a33af3f